scimath/Fitting/FitGaussian.h

```diff
@@ -177,6 +177,10 @@ class FitGaussian
                 T maximumRMS = 1.0, uInt maxiter = 1024, 
                 T convcriteria = 0.0001);
 
+  // Allow access to the fit parameters from this class
+  Matrix<T> solution(){return itsSolutionParameters;};
+  Matrix<T> errors(){return itsSolutionErrors;};
```

Comment:
Probably better to return both as
const Matrix<T>&

Reply:
Agreed